app/views/fragments/instanceTable.scala.html

```diff
@@ -0,0 +1,50 @@
+@(instances: List[Instance], emptyChar: String = "-")
+
+<div class="container">
+@if(instances.nonEmpty) {
+    <div class="ami-usage--message">AMI Usage</div>
```

Comment:
I would add here the instances and launch configuration count. 
Ex: `This AMI is used by 8 instances and 3 launch configurations`. Something like this